### PR TITLE
Fredrik H – Project Happy Thoughts

### PR DESCRIPTION
**Pull request by karlfredrikholm**
Link to my deployed project: https://gladatankar.netlify.app/

My reflections on how this week's project turned out:
Fun project, learned a lot. Got more comfortable with React.

Things I'd like to have clarified or explained in more detail:
Error messages when doing requests to an api. React best practices for structuring you components, props etc.

### CODE REVIEW

File: code/src/components/Feed/Feed.module.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.feedGrid {
+  background-color: whitesmoke;
+  display: grid;
+  grid-template-columns: 1fr;
```
Comment (AgnesAhlman) [inline code/src/components/Feed/Feed.module.css:4]: I don't think you need grid here? or at least I could not se any differens while testing in the inspector

**Review by AgnesAhlman — COMMENTED** [review]
Really well done! Nice comments and I really like the message that pops up if you try to post a too short thought.

File: code/src/components/ThoughtForm/ThoughtForm.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const [thought, setThought] = useState('');
+
+  // for handling when a message is too short to be posted
+  const [tooShortThought, setTooShortThought] = useState('');
+
+  // these things are for the character countdown
+  const maxChars = 140;
+  const [remainingChars, setRemainingChars] = useState(maxChars);
+
+  // Two things had to be handled when the textarea input changes:
```
[the remark this answers is not in the document]
Comment (karlfredrikholm) [reply code/src/components/ThoughtForm/ThoughtForm.jsx:16]: Thanks 🤝 !! Commenting made it easier for me to (make sure that I) understand too 😅

File: code/src/components/ThoughtForm/ThoughtForm.module.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
+.thoughtForm {
+  background-color: rgb(229, 229, 229);
+  border: 1px solid black;
+  display: grid;
+  grid-template-columns: 1fr;
```
[the remark this answers is not in the document]
Comment (karlfredrikholm) [reply code/src/components/ThoughtForm/ThoughtForm.module.css:5]: Nice catch! You're right, it doesn't seem to do anything here 😅

**Pull request closed without merging**